okapi-operation/src/axum_integration/mod.rs

```diff
@@ -35,8 +39,21 @@ use crate::*;
         )
     )
 )]
-pub async fn serve_openapi_spec(spec: State<OpenApi>) -> Json<OpenApi> {
-    Json(spec.0)
+pub async fn serve_openapi_spec(spec: State<OpenApi>, headers: HeaderMap) -> Response {
+    match headers.get("Accept").and_then(|h| h.to_str().ok()) {
```

Comment:
Probably we should accept `*/*` as Accept and return JSON in that case.

Reply:
Fixed